testscripts/decode.py

```diff
@@ -0,0 +1,80 @@
+import os
```

Comment:
think about add header like "#!/usr/bin/python", and change this file to executable